### DISCUSSION TITLE
Connman not setting up in Void Linux

### DISCUSSION THREAD

**Discussion by YamiYukiSenpai** [lxqt/lxqt#2056] [Q&A]
![image](https://user-images.githubusercontent.com/13153124/128413395-6f547a63-aedd-4795-bd41-b22e5f046b19.png)

Both Connman & CMST (along with their dependencies) are installed.

Any idea what I could be missing?

**Comment by tsujan** [comment]
Do you have NetworkManager and if so, did you stop it? If I correctly remember, you don't use systemd; so I can't help you with stopping NetworkManager.

**Reply by YamiYukiSenpai** [reply]
I don't have Network Manager installed, but I did make sure that dhcpcd, and others mentioned in the [Void Handbook](https://docs.voidlinux.org/config/network/connman.html) are disabled.

And Connman is enabled, but CMST won't 
![Uploading Void-Flameshot_2021-08-05_20-15.png…]()
![Void-Flameshot_2021-08-05_20-21](https://user-images.githubusercontent.com/13153124/128456217-d5d7c945-0f9e-43b8-9106-ccbcd3f57a9f.png)

```
$ connmanctl 
Error getting services: Rejected send message, 3 matched rules; type="method_call", sender=":1.49" (uid=1000 pid=3471 comm="connmanctl ") interface="net.connman.Manager" member="GetServices" error name="(unset)" requested_reply="0" destination="net.connman" (uid=0 pid=898 comm="conError getting peers: Rejected send message, 3 matched rules; type="method_call", sender=":1.49" (uid=1000 pid=3471 comm="connmanctl ") interface="net.connman.Manager" member="GetPeers" error name="(unset)" requested_reply="0" destination="net.connman" (uid=0 pid=898 comm="connmand Error getting technologies: Rejected send message, 3 matched rules; type="method_call", sender=":1.49" (uid=1000 pid=3471 comm="connmanctl ") interface="net.connman.Manager" member="GetTechnologies" error name="(unset)" requested_reply="0" destination="net.connman" (uid=0 pid=898 comm="connmand -n ")
connmanctl> agent on
Error registering Agent: Rejected send message, 3 matched rules; type="method_call", sender=":1.49" (uid=1000 pid=3471 comm="connmanctl ") interface="net.connman.Manager" member="RegisterAgent" error name="(unset)" requested_reply="0" destination="net.connman" (uid=0 pid=898 comm="connmand -n ")
connmanctl> services
Error: Rejected send message, 3 matched rules; type="method_call", sender=":1.49" (uid=1000 pid=3471 comm="connmanctl ") interface="net.connman.Manager" member="GetServices" error name="(unset)" requested_reply="0" destination="net.connman" (uid=0 pid=898 comm="connmand -n ")
```

**Reply by tsujan** [reply]
Although various questions can be asked in the Discussions sections, they should be related to LXQt. Your question is about CMST under Void. The correct place for asking it is a Void forum.
